avm/res/network/p2s-vpn-gateway/main.bicep

```
@@ -112,12 +112,12 @@ resource p2sVpnGateway 'Microsoft.Network/p2svpnGateways@2024-01-01' = {
                 '${associatedRouteTableName}'
               )
             }
-            inboundRouteMap: (!empty(inboundRouteMapResourceId))
+            inboundRouteMap: (!empty(inboundRouteMapResourceId ?? ''))
```

```suggestion
            inboundRouteMap: (!empty(inboundRouteMapResourceId))
```
`empty(null)` is false